src/xSTIR/cSTIR/include/sirf/STIR/stir_x.h

```diff
@@ -947,6 +947,16 @@ The actual algorithm is described in
           return this->get_reconstruction_method().get_num_subsets();
         }
 
+        void set_max_scale_factor_value(float v)
+        {
+            set_max_scale_value(v);
+        }
+
+        void set_min_scale_factor_value(float v)
+        {
+            set_min_scale_value(v);
+        }
+
```

Comment:
I prefer to have the Python and C++ names to be identical. As far as I can see, these currently are SIRF names in addition to the (already exposed?) STIR names? Maybe just cut this?

Reply:
STIR's `set_min_scale_value` cannot be called directly by user's code because of
```
    class PETScatterEstimator : private stir::ScatterEstimation
```

Reply:
I think `using stir::ScatterEstimatoin::set_max_scale_value` will work.

Reply:
```
/home/sirfuser/devel/buildVM/sources/SIRF/src/xSTIR/cSTIR/cstir_p.cpp: In function ‘void* sirf::cSTIR_setScatterEstimatorParameter(const DataHandle*, const char*, const DataHandle*)’:
/home/sirfuser/devel/buildVM/sources/SIRF/src/xSTIR/cSTIR/cstir_p.cpp:875:53: error: cannot call member function ‘void stir::ScatterEstimation::set_max_scale_value(float)’ without object
  875 |         stir::ScatterEstimation::set_max_scale_value(dataFromHandle<float>(hv))
```

Reply:
The first way does not work - see the error message I reported above

The second way: where do I put using statements? I tried
```
    class PETScatterEstimator : private stir::ScatterEstimation
    {
      using recon_type = stir::OSMAPOSLReconstruction<DiscretisedDensity<3,float>>;
      using stir::ScatterEstimation::set_max_scale_value;
```
got
```
/home/sirfuser/devel/buildVM/sources/SIRF/src/xSTIR/cSTIR/cstir_p.cpp:880:32: error: ‘void stir::ScatterEstimation::set_max_scale_value(float)’ is inaccessible within this context
  880 |         obj.set_max_scale_value(dataFromHandle<float>(hv));
```

Reply:
The 1st option will work. this is a modification of your own. The lines above need to be in the (`public` section) of the class def.

similarly for `using`. Needs to be after `public`.

Reply:
It turned out I was applying `stir::ScatterEstimation::` prefix in a wrong place (in `cstir_p.cpp`) - all works now, thanks!